modules/core_project_factory/main.tf

```diff
@@ -172,8 +172,8 @@ resource "google_project_services" "project_services_authority" {
 /******************************************
   Shared VPC configuration
  *****************************************/
-resource "google_compute_shared_vpc_service_project" "shared_vpc_attachment" {
-  count = var.shared_vpc != "" ? 1 : 0
+resource "google_compute_shared_vpc_service_project" "shared_vpc_attachment"
+  count = var.shared_vpc_subnets_count != null ? 1 : 0
```

Comment:
does it make sense to create boolean variable for `var.shared_vpc_subnets_count != null` ?

Reply:
Yes it makes sense, I'll try to test it